decidim-core/app/controllers/decidim/scopes_controller.rb

```diff
@@ -13,7 +13,7 @@ def search
                else
                  current_organization.top_scopes
                end
-      root_option = if params[:include_root] && !params[:term].present?
+      root_option = if params[:include_root] && params[:include_root] != "false" && !params[:term].present?
```

Comment:
> Mmm, I have had the same doubt, but I think it is not the same. I want it to enter the if block if params[:include_root] has any value different from nil and "false". The first condition excludes nil and "false" is excluded by the last one.
This case occurs when you go to https://github.com/decidim/decidim/blob/master/decidim-proposals/app/views/decidim/proposals/proposals/_filters.html.erb#L32 and change include_root: true with include_root: false. As this info goes through the querystring, false is received as "false". Even when currently there is no such case now, it happened to me twice, so I think is better this way to avoid wasting time in the future.

What was I thinking? :laughing: You're absolutely right, my bad!

> Regarding #1500 (comment), I think I didn't understand what you were saying. The extra option that appears is a "Loading..." message, that disappears when the AJAX response is received. I didn't changed anything related, maybe there was a problem with AJAX requests on that moment and the 'extra option' didn't disappear at all.

No, I meant to extra search results appearing when you start filtering. Normally you expect that filtering actually reduces the number of options available. But I understand now how this makes sense since showing all subscopes of all levels is not useful (and would overflow) and having a quick selector of first level scopes without searching is nice. So forget everything I said regarding this matter, it's great as it is! :+1: